catalyst/contrib/nn/modules/rms_norm.py

```diff
@@ -0,0 +1,38 @@
+import torch
+import torch.nn as nn
+
+
+class RMSNorm(nn.Module):
+    """
+    An implementation of [RMS Normalization](
+    https://openreview.net/pdf?id=SygkZ3MTJE.
+    RMS Normalization stabilises the training of deep neural networks by
+    normalising the outputs of neurons from a particular layer. It computes:
+    output = (scale * (tensor / (std(tensor) + epsilon)) + bias
+    where std denotes standard-deviation
+    # Parameters
+    dimension : `int`, required.
+    The dimension of the layer output to normalize.
+    epsilon : `float`, optional, (default = 1e-8)
+    An epsilon to prevent dividing by zero in the case
+    the layer has zero variance.
+    is_bias : `bool`, optional, (default = False)
+    A boolean value whether to include bias term while normalization.
+    # Returns
+    The normalized layer output.
```

Comment:
Could you please follow [Google Style Python Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)?

Reply:
Fixed it